core/completion.go

```diff
@@ -318,13 +311,19 @@ func (n *OpenBazaarNode) ReleaseFundsAfterTimeout(contract *pb.RicardianContract
 			if err != nil {
 				return err
 			}
+
 			if confirms < minConfirms {
 				EscrowTimeLockedError = fmt.Errorf("Tx %s needs %d more confirmations before it can be spent", r.Txid, int(minConfirms-confirms))
 				return EscrowTimeLockedError
 			}
+
 			outpoint := wire.NewOutPoint(hash, r.Index)
-			utxo.Op = *outpoint
-			utxos = append(utxos, utxo)
+			var txInput = wallet.TransactionInput{
+				Value:         r.Value,
+				OutpointIndex: outpoint.Index,
+				OutpointHash:  outpoint.Hash.CloneBytes(),
+			}
```

Comment:
@cpacia Do we need to specify the `LinkedAddress` here? Whose?

Reply:
Yes I believe so. Should be in the TransactionRecord obj after we migrate it.

Reply:
Fixed.